src/styles/less/style.less

```diff
@@ -1,6 +1,6 @@
 @root-entry-name: default;
 
-@import '~antd/lib/style/components.less';
+@import '~antd/dist/antd.less';
```

Comment:
@quy-truong It's not removed, it moves into the style itself.

Reply:
IIRC, there is an issue with using `antd.less` directly 🤔